deploy/addons/freshpod/freshpod-dp.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
can we make this a ReplicaSet, I believe we don't need Deployment’s rollout/rollback semantics here because add-on manager manages it exclusively.

Reply:
Yes, I have been changed to RC because other add-ons also use this kind of resource.